src/q7/radiation-ELEC2795/exam/2019/Janvier/All/radiation-ELEC2795-exam-2019-Janvier-All.tex

```diff
@@ -0,0 +1,182 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\usepackage{../../../../../../eplunits}
+
+\hypertitle{Radiation and communication systems}{7}{ELEC}{2795}{2019}{Janvier}{All}
+{Alice Borbath \and Olivier Leblanc \and Maxime Wattiaux}
+{Cristophe Craeye, Danielle Janvier, Jérome Louveaux, Claude Oestges and Luc Vandendorpe}
+
+\section{(4,5 points)}
+
+Frequency: $f=\SI{75}{GHz}$.
+
+\begin{figure}[ht!]
+    \centering
+    \includegraphics[scale=0.4]{Q1.png}
+\end{figure}
+
+\begin{enumerate}
+    \item (1 point) Calculate the reflection coefficient at $z=0$.
+    \item (1 point) Calculate the smallest value of $L$ such that there is total transmission at $z=0$ (in mm).
+    \item (2,5 points) Assume that the frequency is not exactly at \SI{75}{GHz}, calculate the percentage of reflected power.
+    Consider the frequency equal to $f=f_0(1+\delta)$ and develop to the first order of $\delta$.
+    
+    Hint: use $\sin(\pi+\epsilon)=-\sin(\epsilon)\approx-\epsilon$ and $\cos(\pi+\epsilon) \approx -1$.
+\end{enumerate}
+
+\nosolution
+
+\section{(4,5 points)}
+
+A wireless horizontal link is established over 100 meter distance at \SI{2.4}{GHz}. In first instance, we will assume that the antennas are placed high enough to be able to omit the effect of interaction with the ground. On the transmit side, the antenna is actually made of an array of two parallel half-wave dipoles, oriented perpendicularly to the ground and space by half a wavelength. The power fed to the pair of antennas is assumed to be split equally between the antennas, and matching at the common feeding part is supposed perfect. On the receiving side, an antenna with circular polarization, with 10dB gain is used. The antenna is correctly pointed and its pattern falls of by 3 dB at an angle close to 25 degrees from boresight. That antenna is badly matched: it has an impedance of $100-20j$ $\Omega$, while the receiver has \SI{50}{\Omega} input impedance. Its noise figure is also relatively bad, corresponding to 4 dB. The transmitted array is not well pointed towards the receiving antenna, because of non-equal phases in the two excitations, such that the pattern is actually pointing 20 degrees away from the line of sight. The noise contributed by the environment will be calculated assuming a \SI{290}{K} temperature in all directions and the occupied bandwidth is \SI{20}{MHz}.
```

Comment:
```suggestion
A wireless horizontal link is established over a 100 meter distance at \SI{2.4}{GHz}. In first instance, we will assume that the antennas are placed high enough to be able to omit the effect of interaction with the ground. On the transmit side, the antenna is actually made of an array of two parallel half-wave dipoles, oriented perpendicularly to the ground and spaced by half a wavelength. The power fed to the pair of antennas is assumed to be split equally between the antennas, and the matching at the common feeding part is supposed perfect. On the receiving side, an antenna with circular polarization, with a 10 dB gain is used. The antenna is correctly pointed and its pattern falls off by 3 dB at an angle close to 25 degrees from boresight. That antenna is badly matched: it has an impedance of $100-20\imagj \Omega$, while the receiver has \SI{50}{\Omega} input impedance. Its noise figure is also relatively bad, corresponding to 4 dB. The transmitted array is not well pointed towards the receiving antenna, because of non-equal phases in the two excitations, such that the pattern is actually pointing 20 degrees away from the line of sight. The noise contributed by the environment will be calculated assuming a \SI{290}{K} temperature in all directions and the occupied bandwidth is \SI{20}{MHz}.
```